src/bespokelabs/curator/request_processor/base_request_processor.py

```diff
@@ -76,6 +77,51 @@ def run(
         """
         pass
 
+    def _get_bad_cache_files(self, working_dir: str, dataset: Optional[Dataset]) -> List[int]:
```

Comment:
small nit: 

can we rename this to _verify_existing_request_files

or something that is specific to request files (because there are a bunch of other types of files in the cache) and it is unclear what `bad` is.